### DISCUSSION TITLE
Tinkers' construct mod support

### DISCUSSION THREAD

**Discussion by Omgise** [Roadhog360/Et-Futurum-Requiem#176] [Ideas]
Adds additional tools and weapons for Tinkers' Construct

**Comment by Roadhog360** [comment]
There are already other addons for Tinkers' Contruct which adds support for the materials in Et Futurum Requiem.

**Reply by Omgise** [reply]
> There are already other addons for Tinkers' Contruct which adds support for the materials in Et Futurum Requiem.

Thank you for your reply. I think I have found the mod.

**Comment by Roadhog360** [comment]
NetheritePlus+ has been discontinued and I have been granted access to its source code, I am removing the "Won't Add" label as I want to merge its TCon integration soon, alongside its various features.
